### PR TITLE
support show collation

### PR DESCRIPTION
**Pull request by siddontang**

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
LGTM

File: parser/parser.y [hunk 1]
```diff
@@ -3095,6 +3096,26 @@ ShowStmt:
 			Where: expression.Expr($5),
 		}
 	}
+|	"SHOW" "COLLATION"
```
Comment (ngaut) [inline parser/parser.y]: Are there any S/R conflicts ?

Comment (siddontang) [inline parser/parser.y]: no

Comment (siddontang) [inline parser/parser.y]: make parser
```
go get github.com/qiuyesuifeng/goyacc
go get github.com/qiuyesuifeng/golex
a=`mktemp temp.XXXXXX`; \
	goyacc -o /dev/null -xegen $a parser/parser.y; \
	goyacc -o parser/parser.go -xe $a parser/parser.y; \
	rm -f $a; \
	rm -f y.output
Parse table entries: 190529 of 516708, x 16 bits == 381058 bytes
Parse table entries: 190529 of 516708, x 16 bits == 381058 bytes
golex -o parser/scanner.go parser/scanner.l
```

**Comment by shenli** [comment]
LGTM

**Pull request merged**